test/sagas/collection_test.js

```diff
@@ -101,14 +99,27 @@ describe("collection sagas", () => {
   });
 
   describe("createRecord()", () => {
-    describe("Success", () => {
-      let collection, createRecord;
+    let collection;
+
+    before(() => {
+      collection = {createRecord() {}};
+      const bucket = {collection() {return collection;}};
+      setClient({bucket() {return bucket;}});
+    });
```

Comment:
It's just as we don't stub anything here, we don't have to reset/revert anything too, so `before` is just enough.